src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int indexOfColor = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[indexOfColor];
+        return color.toString();
```

Comment:
Avoid creating a new instance of Random every time the method is called. Instead, create a single instance as a class-level variable and reuse it.